ext/reflection/tests/ReflectionAttribute_toString.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+ReflectionAttribute::__toString
+--FILE--
+<?php
+
+#[Foo, Bar(a: "foo", b: 1234), Baz("foo", 1234)]
+function foo() {}
+
+$refl = new ReflectionFunction('foo');
+echo $refl->getAttributes()[0];
+echo $refl->getAttributes()[1];
+echo $refl->getAttributes()[2];
+--EXPECTF--
+Attribute [ Foo ]
+Attribute [ Bar ] {
+  - Arguments [2] {
+    Argument #0 [ a = 'foo' ]
+    Argument #1 [ b = 1234 ]
+  }
+}
+Attribute [ Baz ] {
```

Comment:
This looks okay, but I wonder if it would make sense to print it as
```
Attribute [ Baz('foo', 1234)` ]
```
or even
```
#[Baz('foo', 1234)]
```
instead?

Maybe this doesn't compose well when included in other reflection output though.

Reply:
Yes, the idea was that this gets its own `_attribute_string` at some point and nests into the existing reflection structures that look this way.